### PR TITLE
Feat: Protected Areas

### PR DESCRIPTION
**Pull request by geriux**
- New Map Modal wrapper to use for different components
- Protected Areas component (WIP)
- Added splash screen for iOS

screen shot 2017-02-07 at 18 52 46
screen shot 2017-02-07 at 18 52 06

TODO: Finish map wrapper to receive texts properties for title and legend, button when selecting a protected area.

### CODE REVIEW

File: app/components/setup/boundaries/index.js [hunk 1]
```diff
@@ -62,16 +92,31 @@ class SetupBoundaries extends Component {
             </TouchableHighlight>
           </View>
         </View>
-        <ProtectedAreas
-          iso={'VEN'}
-          visible={this.state.showProtectedAreas}
-        />
+        {this.state.showProtectedAreas &&
```
Comment (j8seangel) [inline app/components/setup/boundaries/index.js]: As we have 3 components in the setup slider I've used the flag to hide the Areas map instead the modal visible prop to try to keep it as light as possible

File: app/components/setup/protected-areas/styles.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import { StyleSheet } from 'react-native';
+
+export default StyleSheet.create({
+  map: {
+    ...StyleSheet.absoluteFillObject
```
Comment (j8seangel) [inline app/components/setup/protected-areas/styles.js]: 🥇

**Comment by j8seangel** [comment]
Set up workflow completed !!! 🎉 
- Country and protected area selection
- Custom polygon creation
- Areas of interest snapshots (AOI)
- Saved AOI into the API.

fw-setup

**Comment by geriux** [comment]
Yay!! Team work! 🙌 🙌 🙌 🙌 🙌

**Comment by j8seangel** [comment]
🙏  Remember to run `npm install` to use the new `react-native-maps` fork library.

**Pull request merged**